2022/incoming/the-christmas-time-machine.pod

```diff
@@ -0,0 +1,81 @@
+Author: Thibault DUPONCHELLE <thibault.duponchelle@gmail.com>
+Title: The Christmas time machine
+Topic: Feature::Compat::Try
+
+=encoding utf8
+
+=head1 The Christmas time machine
+
+=head2 Terrible flashback at the line work
+
+At North Pole, elves are just starting the Christmas 2022 preparations.
+
+As usual they restarted doing line work, building toys, then sorting them per area. Everyone happily singing and working! When suddenly one elf put the first toy in a box marked "Salt Lake City" and got surprised it was not empty...
+
+And he got a terrible flashbak, they seemed to have forgotten to deliver 2021 toys in Salt Lake City!
+
+=head2 Go ask Santa
+
+The elves were panicked, hence they reached quickly Santa for a solution. 
+
+He took some time to think about it then had an idea: Let's use the time machine to come back in 2021 and deliver the gifts!
+
+(if you wonder: YES, time machines do exist in North Pole! And I bet that elves even already used it in the past for similar mistakes? :P)
+
+So a group of Elves decided to use the time machine, fixed the arrival date to 25 December 2021, pushed the "go" button and zooooooooooooop!
+
+=head2 Try to deliver back in the past
+
+When arrived in 2021, they immediately tried to run Perl software that they took with them.
+
+For the record, it's Perl code like this:
+
+=begin perl
+
+	#!/usr/bin/perl
+
+	use feature 'try';
+	no warnings 'experimental::try';
+
+	try {
+	    deliver();
+	} catch($e) {
+	    print "Failed to deliver\n";
+	}
+
+=end perl
+
+But it failed with:
+
+	Feature "try" is not supported by Perl 5.30.0 at deliver.pl line 3.
```

Comment:
Slight continuity problem... 5.30 was released in 2019. Would they not have 5.34 in 2021?

Reply:
I used the one that is shipped with Ubuntu in 2020 

Perl 5.34 has already the `try` feature...